spec/controllers/tours_controller_spec.rb

```diff
@@ -1,5 +1,55 @@
 require 'rails_helper'
 
 RSpec.describe ToursController, type: :controller do
+  let(:user) { create :staff_user }
 
+  context "User authenticated" do
```

Comment:
This is great. A common patten though would be to say:

```ruby
let :params do
  {
    title: "Awesome Tour",
    staff_user_id: user.id,
    cultural_center_id: user.cultural_center.id
  }
end

let :invalid_params do
  params.merge(title: "")
end
```

Then, just use those in your tests.